packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -467,8 +467,21 @@ class HeroController extends NavigatorObserver {
     }
   }
 
+  // Suspend Hero animations while a user gesture is controlling the navigation.
```

Comment:
if i'm reading this right it's not suspending active animations, it's preventing new ones from starting. Is that what you meant? If so I'd call this _questsEnabled rather than _suspendQuest (and flip the meaning of the boolean)